cfme/tests/configure/test_access_control.py

```diff
@@ -1926,3 +1926,26 @@ def test_tenant_visibility_miq_ae_namespaces_all_parents():
         startsin: 5.5
     """
     pass
+
+
+@pytest.mark.manual
+@test_requirements.cfme_tenancy
+@pytest.mark.tier(1)
+def test_cloud_tenant_crud_rhos():
+    """
+    Polarion:
+        assignee: mnadeem
+        casecomponent: Configuration
+        caseimportance: medium
+        initialEstimate: 1/2h
+        tags: cfme_tenancy
+        testSteps:
+            1. Add RHOS provider and perform refresh
+            2. Navigate to Compute -> Clouds -> Providers
+            3. Click on RHOS provider which was added
+            4. In the Relationships table click on Cloud Tenants
+            5. Configuration ->Create Cloud tenant
+            6. Edit created tenant
+            7. Try to delete it
```

Comment:
**Required** For manual only test cases, we should have `testSteps` and `expectedResults`

Reply:
I think its optional... why we need expected result for each step?

Reply:
@mnadeem92 can you please help me here? Actually, this assigned to me but need to move you.

Reply:
expectedResult seems optional here as test step already mentioned to execute the CRUD (create/edit and delete) operation, but if you want to include it then expectedResult would be something like "CRUD operation should executed successfully".

Reply:
Yes, manual test cases should have testSteps and expectedResults.

What do you expect to see after the entity is edited? what fields should be edited? should those be reflected immediately, or after some delay? Should the changes be verified anywhere else in the UI?

After delete, should the entity's removal be verified from anywhere else in the UI? Should the same page just be refreshed?

Reply:
This has already been automated: https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/cloud/test_tenant.py

Reply:
@mshriver thanks for pointing this. :+1: removed

@mshriver still not agreed with you... about expectedResult required. As per me totally optional and its should contributes privilege. How he or she want to write own test. 

Lets take step is "Add provider"
In some contributor mind provider successfully added some think about data collected or not. In this case expected result he must added so other can easy understand. 

Its not possible to collect all observations. Yup good to add expected results but its should optional not strict like "delete provider" is step and contributes need to write "deleted or not".

expectedResults indicate tabular form but same thing can achieve by simple single line statement.

Reply:
@digitronik There does not need to be a result for every test step, some can be left blank. For example:
https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/cloud/test_tag_mapping.py#L57

I believe it should be explicitly stated in `expectedResults` on what the tester is looking for in regards to the test. It also creates consistency across all the manual tests

Reply:
I'm not expecting the test author to "collect all observations" - but there should be some expected result of the test step. As @jawatts has pointed out not all expectedResults lines have to be populated, and we've certainly accepted many PR's with only an expectedResult on the last test step.

I don't plan on being very strict with non-CFME contributors to integration_tests, that want to define some manual test. As long as their docblock is thorough enough, we can push commits that fill out the polarion metadata relevant to the test.

Reply:
@jawatts  that what I speak... optional, If expectedResults steps are optional means expectedResult tag is also optional for manual or automated.

Above test was just curd... so add/edit/delete operation. I won't find anything required part of expectedResult. I was not even the owner of test just moving as it was with my name with proper FA owner permission. 

Yup, we can ask for the expected result when contributor statement not clear. I don't think force is a better way as creating a test is the owner right. Every contributor has own way of testing.